src/ISA.json

```diff
@@ -1948,7 +1948,7 @@
     },
     {
       "mnemonic": "sha256sum0",
-      "name": "TBD",
+      "name": "SHA2-256 Sum0 ",
```

Comment:
extra space after "Sum0". Everything else looks good.
Fix this, then feel free to squash the commits into a single commit and the force-push to final changes. I can do this when I merge, if you prefer.